### PR TITLE
PulsePoint adapter - Video support + userId module support

### PR DESCRIPTION
**Pull request by anand-venkatraman**
- [x] Feature

## Description of change
1) Video InStream and Outstream support
2) Support for openrtb paramters - bidfloor/badv/bcat/battr
3) Support for user id modules.

- avenkatraman@pulsepoint.com

## Other information
Test pages below 
[InStream](http://projects.contextweb.com/av/pb-video/instream/)
[OutStream](http://projects.contextweb.com/av/pb-video/outstream/)
[User Ids](http://projects.contextweb.com/av/pb-user/)

### CODE REVIEW

**Comment by robertrmartinez** [comment]
Will take a look now thank you

**Comment by anand-venkatraman** [comment]
@robertrmartinez - did you get a chance to look into this?

File: modules/pulsepointBidAdapter.md [hunk 1]
```diff
@@ -40,5 +40,57 @@ Please use ```pulsepoint``` as the bidder code.
...
+        video: {
+            playerSize: [640, 480],
+            context: 'outstream'
+        }
+      },
+      bids: [{
+          bidder: 'pulsepoint',
+          params: { 
+              cp: 512379,
+              ct: 505642
```
Comment (robertrmartinez) [inline modules/pulsepointBidAdapter.md]: Missing a comma here

Comment (anand-venkatraman) [reply modules/pulsepointBidAdapter.md]: fixed

**Comment by anand-venkatraman** [comment]
Not sure why you didnt get a bid, but here are the test pages for outstream and instream. Please give it a go.
http://projects.contextweb.com/av/pb-video/instream/
http://projects.contextweb.com/av/pb-video/outstream/slider.html

**Pull request merged**